azavea/pfb-network-connectivity
# Don't display FUNCTIONAL=path if its a crosswalk

## Overview

Skips display of any bike paths that are also tagged as crosswalks. This cleans up a bunch of the small, odd "quotation-mark" paths that appear all over the place.


### Demo

#### Before

![](https://user-images.githubusercontent.com/1818302/27971347-610eae7a-6320-11e7-80e3-9d72373840b3.png)

#### After

<img width="782" alt="screen shot 2017-07-14 at 10 32 27" src="https://user-images.githubusercontent.com/1818302/28216935-63ad03b4-6881-11e7-8234-84c73dccb717.png">

## Testing Instructions

 * Rebuild analysis and tilemaker container to ensure changes from #530 are applied
 * Run analysis + tilemaker jobs and inspect output on place detail view

Closes #519
